src/util.ts

```diff
@@ -50,47 +47,44 @@ export function mapUrlPathToBucketPath(
   url: URL,
   env: Pick<Env, 'DIRECTORY_LISTING'>
 ): string | undefined {
-  const urlToBucketPathMap: Record<string, string> = {
-    dist: DIST_PATH_PREFIX + (url.pathname.substring('/dist'.length) || '/'),
-    download:
-      DOWNLOAD_PATH_PREFIX +
-      (url.pathname.substring('/download'.length) || '/'),
-    docs: DOCS_PATH_PREFIX + (url.pathname.substring('/docs'.length) || '/'),
-    api: API_PATH_PREFIX + (url.pathname.substring('/api'.length) || '/'),
-    metrics: url.pathname.substring(1), // substring to cut off the /
-  };
-
-  // Example: /docs/asd/123
-  let bucketPath: string | undefined;
-
   const splitPath = url.pathname.split('/'); // ['', 'docs', 'asd', '123']
```

Comment:
BTW what if you do:

const [, basePath, ...pathPieces] = url.pathname.split('/');

Then you wouldn't need to slice it below, right?